packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -78,6 +78,11 @@ const propTypes = {
    * If set to true, the AbstractModal is rendered inside a NotificationDialog.
    */
   isCalledFromNotificationDialog: PropTypes.bool,
+  /**
+   * @private
+   * If set to true, then the focus lock will get enabled.
+   */
+  enableFocusLock: PropTypes.bool,
```

Comment:
In what scenario would we not want the focus lock to be enabled?  Should this really be a configurable property?

Reply:
This was not suppose to be private prop @AH106586Harika , We are suppose to create this as class prop on modal manager as well. Fusion team is consuming modal manager for pass-through and fusion component has a prop called `shouldTrapFocus` to enable focus trap ( by default focusTrap is not enabled ). In order to address this GAP modal manager should also have similar prop to enable focus trap when it is required.

Reply:
Added as suggested.